torch/nested/_internal/ops.py

```diff
@@ -472,6 +472,31 @@ def prod_dim_int(func, *args, **kwargs):
     return NestedTensor(func(inp._values, **new_kwargs), **extract_kwargs(args[0]))
 
 
+@register_jagged_func(
+    torch.ops.aten.split.Tensor, "self: jt, split_size: any, dim: any"
+)
+def split_tensor(func, *args, **kwargs):
+    _, new_kwargs = normalize_function(
+        func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
+    )
+
+    inp = new_kwargs.pop("input")
+    values = inp._values
+
+    # hack to split on the last dim
+    dim = new_kwargs["dim"]
```

Comment:
You can use `new_kwargs["dim"] = _wrap_jagged_dim(inp.ndim(), new_kwargs["dim"], "split")` to normalize the dim.